src/components/Homepage/index.js

```diff
@@ -2,8 +2,8 @@ import React, { useState } from "react";
 import { Grid, Typography, IconButton } from "@mui/material";
 import { useFormik } from "formik";
 import { ReactComponent as Logo } from "../../images/Logo.svg";
-import { ReactComponent as VFLogo } from "../../images/VFLogo.svg";
-import { ReactComponent as Felix } from "../../images/Felix.svg";
+import Felix from "../../images/Felix.svg";
+import VFLogo from "../../images/VFLogo.svg";
```

Comment:
Why remove the svg import as ReactComponent? If you import an svg as a component you are basically injecting the SVG code into your HTML and that gives you all the power to manipulate an svg file. Power like animating, changing width/height and color etc